test/e2e/framework/util.go

```diff
@@ -2847,7 +2857,13 @@ func RemoveLabelOffNode(c *client.Client, nodeName string, labelKey string) {
 		return
 	}
 	delete(node.Labels, labelKey)
-	nodeUpdated, err := c.Nodes().Update(node)
+	var nodeUpdated *api.Node
+	for attempt := 0; attempt < UpdateRetries; attempt++ {
+		nodeUpdated, err = c.Nodes().Update(node)
```

Comment:
probably should re-get node when retry, since node is updated by someone else and node.ResourceVersion is out of date.